tap_github/streams.py

```diff
@@ -266,6 +267,34 @@ class CommunityProfileStream(GitHubStream):
     ).to_dict()
 
 
+class LanguagesStream(GitHubStream):
+    name = "languages"
+    path = "/repos/{org}/{repo}/languages"
+    primary_keys = ["repo", "org", "language_name"]
+    parent_stream_type = RepositoryStream
+    ignore_parent_replication_key = False
+    state_partitioning_keys = ["repo", "org"]
+
+    def parse_response(self, response: requests.Response) -> Iterable[dict]:
+        """Parse the language response and reformat to return as an iterator of [{language_name: Python, bytes: 23}]."""
+        if response.status_code in self.tolerated_http_errors:
+            return []
+
+        languages_json = response.json()
+        for key, value in languages_json.items()
+            yield {"language_name": key, "bytes": value}
```

Comment:
Does this just return those languages in use or is it one row per item of everything here?:

https://github.com/github/linguist/blob/master/lib/linguist/languages.yml

If the latter (nulls and zeros are present), we may want to do something like:

```suggestion
            if value:
                yield {"language_name": key, "bytes": value}
```

Reply:
No, the tests we did on the endpoint only list the languages actually used in the repo, so it should be fine without filtering.